### ISSUE TITLE
[feature]auto select qbittorrent server based on ssid or ip

### ISSUE THREAD

**Issue by WanderMax** [Bartuzen/qBitController#136]
there are two ways to connect web server: local network or public network via VPN.
for local network, webui address and network ip address should share same ip prefix, like 192.168..., or connect to home WiFi with wifi SSID name. this can be used to auto choose server.

**Comment by Bartuzen** [comment]
What you're asking is when you open the app, a different server will be shown by default depending on whether you are connected to a VPN or not. Alternatively, the default server will be selected based on the Wi-Fi network name. Did I understand correctly?

**Comment by WanderMax** [comment]
yes. when at home, connected to specified SSID, app should auto select local server. this idea comes from tremotesf
